app/uk/gov/hmrc/voapropertylinking/services/PropertyLinkingService.scala

```diff
@@ -76,8 +77,8 @@ class PropertyLinkingService @Inject()(
     propertyLinksConnector
       .getMyOrganisationsPropertyLinks(GetMyOrganisationPropertyLinksParameters(), None)
```

Comment:
It seems odd that this connector call returns "Future[Option[PropertyLinksWithAgents]]" rather than "Future[PropertyLinksWithAgents]". I guess this is what you were asking about on Monday?

Reply:
Yes, I am using existing one now, it is temporally, once new modernised End Point is ready for Property Links Count , I will point to new End point

Reply:
discussing offline